### DISCUSSION TITLE
Generating a poster with Beamer

### DISCUSSION THREAD

**Discussion by MattF-NSIDC** [quarto-dev/quarto-cli#5819] [Q&A]
### Description

Is there any way to generate a Beamer poster with Quarto? Currently the docs are geared towards slides, and I don't see a documented way to use a named Beamer template like `tikzposter` or `beamerposter`, or to provide your own.

**Comment by mcanouil** [comment]
See #2205 for previous discussion on the matter.
(Closing this as duplicate.)

**Reply by MattF-NSIDC** [reply]
Ah, didn't see that because I accidentally searched only in "Feature requests". Apologies.

**Reply by mcanouil** [reply]
No apologies needed 😉

**Reply by MattF-NSIDC** [reply]
Then thanks for the link ;)
